docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -77,70 +90,122 @@ feedbackIntegration({
+| `--success-foreground`         | `successForeground`       | `#268d75`                                 | `#2da98c`                                 | Color used for success-related components (such as text color when feedback is submitted successfully). |
+| `--error-foreground`           | `errorForeground`         | `#df3338`                                 | `#f55459`                                 | Color used for error related components (such as text color when there's an error submitting feedback). |
+| `--background`                 | `background`              | `#ffffff`                                 | `#29232f`                                 | Background color of the widget (injected button and form).                                              |
+| `--border`                     | `border`                  | `1.5px solid rgba(41, 35, 47, 0.13)`      | `1.5px solid rgba(235, 230, 239, 0.5)`    | The border style used for the widget (injected button and form).                                        |
+| `--box-shadow`                 | `boxShadow`               | `0px 4px 24px 0px rgba(43, 34, 51, 0.12)` | `0px 4px 24px 0px rgba(43, 34, 51, 0.12)` | Box shadow style used for the widget (injected button and form).                                        |
+| `--input-foreground`           | `inputForeground`         | `inherit`                                 | `inherit`                                 | Foreground color for form inputs.                                                                       |
+| `--input-background`           | `inputBackground`         | `inherit`                                 | `inherit`                                 | Background color for form inputs.                                                                       |
+| `--input-background-focus`     | `inputBackgroundFocus`    | `inherit`                                 | `inherit`                                 | Background color for form inputs when focused.                                                          |
+| `--input-border`               | `inputBorder`             | `var(--border)`                           | `var(--border)`                           | Border styles for form inputs.                                                                          |
+| `--input-border-radius`        | `inputBorderRadius`       | `6px`                                     | `6px`                                     | Border radius style for form content (such as inputs and buttons).                                      |
+| `--input-outline-focus`        | `inputOutlineFocus`       | `1px auto rgba(108, 95, 199, 1)`          | `1px auto rgba(108, 95, 199, 1)`          | Outline color for form inputs when focused.                                                             |
```

Comment:
Since it's not just the color anymore
```suggestion
| `--input-outline-focus`        | `inputOutlineFocus`       | `1px auto rgba(108, 95, 199, 1)`          | `1px auto rgba(108, 95, 199, 1)`          | Outline for form inputs when focused.                                                             |
```